app/assets/javascripts/rollbar.ready.js

```diff
@@ -0,0 +1,9 @@
+var _rollbarConfig = {
+      endpoint: RollbarConfig.js_endpoint,
```

Comment:
Could you clean up the formatting in this file? 2-space indents, fields aligned in same column, closing brackets in same column as the start of the key.